### ISSUE TITLE
Cant Download UAV Round 2 Training Set

### ISSUE THREAD

**Issue by gunnxx** [MetaSLAM/GPR_Competition#17]
Somehow I can't download the training dataset for UAV Round 2. I use this [dropbox link](https://www.dropbox.com/scl/fo/saejbf9qanbfq40k8jo18/h?dl=0&rlkey=l19kx1vzzahifv3n5lkg19n84). I could click the download, it progressed, but at the end of the download, it was suddenly failing. I tried multiple times using Firefox and Chrome. I have no problem with the validation and testing dataset.

**Comment by ryanzhao9459** [comment]
I tried on my own device, and it worked when I downloaded the training, val, test set individually. Hope this can help u with the problem.

**Comment by gunnxx** [comment]
Yup, turns out cookies problems on all of my browsers.
